### ISSUE TITLE
Accent color not respected in classic theme on app startup

### ISSUE THREAD

**Issue by sviperz** [telegramdesktop/tdesktop#24930]
### Steps to reproduce

1. Go to _Settings > Chat Settings_.
2. Set the "Classic" theme and select a different accent color from the default.
3. Exit the app.
4. Reopen the app.
5. The app will have the classic theme, but the accent color is the default blue and not the one set before.

It was reported also on bugs platform - https://bugs.telegram.org/c/15908

### Expected behaviour

The accent color set previously is applied on app startup.

### Actual behaviour

The accent color resets to default blue, so the app doesn't respect user settings for the default theme.

### Operating system

Pop!_OS 22.04 LTS x86_64

### Version of Telegram Desktop

Telegram Desktop for Linux v4.1

### Installation source

Static binary from official website

### Logs

```shell
From the debug logs upon startup:

[12:20:39.046 00-0000072] Theme: initial load (night: [FALSE]), key_legacy: 0, key_day: 0, key_night: 0 (localstorage.cpp : 25)
```

**Comment by k-kruglov** [comment]
Happens on Windows 10, 4.2.4. x64.

**Comment by sviperz** [comment]
Why is this closed? It is still reproducible every time on the latest **4.8.3**.

**Comment by sviperz** [comment]
This is still reproducible every time on the latest **4.12.2**.

**Comment by sviperz** [comment]
Still reproducible every time on the latest version.

**Comment by sviperz** [comment]
Nope, still reproducible on the latest v5.9.

**Comment by softwarepagan** [comment]
I have this issue on Linux Mint 22 as well. Every time.

**Comment by sviperz** [comment]
Nope, still reproducible on the latest version. Maybe it's time to just fix it.

**Comment by sviperz** [comment]
Still reproducible on the latest version.
